packages/perseus/src/widgets/categorizer/categorizer.tsx

```diff
@@ -53,14 +53,6 @@ export class Categorizer
         uniqueId: _.uniqueId("perseus_radio_"),
     };
 
-    /**
-     * TODO: remove this when everything is pulling from Renderer state
-     * @deprecated get user input from Renderer state
-     */
-    getUserInput(): PerseusCategorizerUserInput {
-        return this.props.userInput;
-    }
```

Comment:
User input is now stored in the parent and we no longer access it through widgets' `getUserInput`